jaide/core.py

```diff
@@ -250,12 +250,15 @@ def commit(self, commands="", confirmed=None, comment=None,
         # passed, use 'annotate system' to make a blank commit
         if not commands:
             commands = 'annotate system ""'
-        clean_cmds = []
-        for cmd in clean_lines(commands):
-            clean_cmds.append(cmd)
+        if config_format == "text":
```

Comment:
could the action also affect whether we should clean the lines or not? Is action being used for either "set" or "stanza"?